CHANGELOG.md

```diff
@@ -36,7 +36,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #1560, Volatile RPC called with GET now returns 405 Method not Allowed instead of 500 - @wolfgangwalther
  - #1604, Change the default logging level to `log-level=error`. Only requests with a status greater or equal than 500 will be logged. If you wish to go back to the previous behaviour and log all the requests, use `log-level=info` - @steve-chavez
  - #1617, Dropped support for PostgreSQL 9.4 - @wolfgangwalther
-
+ - #1854, Dropped undocumented support for gzip compression (which was surprisingly slow and easily enabled by mistake). In some use-cases this makes Postgres up to 3x faster - @aljungberg
```

Comment:
We're not updating the changelog manually anymore, but will generate this from the commit for the next release. Just make sure to give the commit either:

- a prefix of `fix: ...`
- or a prefix of `feat: ...`
- and/or a `BREAKING CHANGE: ...` anywhere in the body

And it will be picked up automatically.

Although I am not sure where I would put this right now... :D

Reply:
`fix` plus `BREAKING CHANGE` would do, I think.

I actually been wanting the changelog, once https://github.com/PostgREST/postgrest/pull/1805 is done, we should generate it on every push to master.